backend/dataall/base/aws/cognito.py

```diff
@@ -15,8 +15,6 @@ def __init__(self):
     def get_user_emailids_from_group(self, groupName):
         try:
             envname = os.getenv('envname', 'local')
-            if envname == 'local' or envname == 'dkrcompose':
```

Comment:
@dlpzx , after removing, while working on local development environment the cognito calls do fail. In that case, do you maybe want to keep this if condition to check if its local dev environment and then raise an Exception ?

Reply:
Good catch! Let me review because I have not deployed it locally in a while

Reply:
I think we need to add it to both places but as a conditional. Let me explain: in the local development we might use the account with a Cognito user pool with local users and we might want to test SES email if we have the right set up in the AWS account. 
- For Cognito local setup, developers will need to create the SSM parameter for the dkrcompose environment and make sure it points at a Cognito user pool with users and groups used in the local deployment. The use case for this will be to test Cognito boto3 calls and the code that handles the response in data.all
- For SES we just need to perform the preparation as if it was a deployment to AWS. I see the point of testing but I also see the scenario where developers have that particular config.json and do not have SES in their development accounts. That's why I added a similar block

Reply:
I tested it and committed it [here](https://github.com/awslabs/aws-dataall/pull/822/commits/4e9a05b21d91f2959b4e5f8e2ca25d57a56bb261)